examples/web-server/nginx/README.md

```diff
@@ -0,0 +1,148 @@
+# How to use NGINX as a reverse-proxy with LetsEncrypt
+
+## Requirements
+
+1. Start a Coder deployment with a wildcard subdomain. See [this guide](https://coder.com/docs/v2/latest/admin/configure#wildcard-access-url) for more information.
+
+2. Configure your DNS provider to point your YOUR_SUBDOMAIN and \*.YOUR_SUBDOMAIN to your server's public ip.
+
+   > For example, to use `coder.example.com` as your subdomain, configure `coder.example.com` and `*.coder.example.com` to point to your server's public ip. This can be done by adding A records in your DNS provider's dashboard.
+
+3. Install NGINX (assuming you're on Debian/Ubuntu):
+
+   ```console
+   sudo apt install nginx
+   ```
+
+4. Stop NGINX service:
+
+   ```console
+   sudo systemctl stop nginx
+   ```
+
+## Adding Coder deployment subdomain
+
+> This example assumes Coder is running locally on `127.0.0.1:3000` for the subdomain `YOUR_SUBDOMAIN` e.g. `coder.example.com`.
+
+1. Create NGINX configuration for this app:
+
+   ```console
+   sudo touch /etc/nginx/sites-available/YOUR_SUBDOMAIN
+   ```
+
+2. Activate this file:
+
+   ```console
+   sudo ln -s /etc/nginx/sites-available/YOUR_SUBDOMAIN /etc/nginx/sites-enabled/YOUR_SUBDOMAIN
+   ```
+
+## Install and configure LetsEncrypt Certbot
+
+1. Install LetsEncrypt Certbot: Refer to the [CertBot documentation](https://certbot.eff.org/instructions?ws=other&os=pip&tab=wildcard)
```

Comment:
When following the certbot docs, it had some options to auto-configure NGINX for me. Should we mention this is not necessary for this guide since we'll do it later?

Reply:
```suggestion
1. Install LetsEncrypt Certbot: Refer to the [CertBot documentation](https://certbot.eff.org/instructions?ws=other&os=pip&tab=wildcard). Be sure to pick the wildcard tab and select your DNS provider for instructions to install the necessary DNS plugin.
```

Reply:
Yes, it is better to mention wildcard here explicitly. Good suggestion.